<ui version="4.0" >
 <class>QualityMapperDialogClass</class>
 <widget class="QDockWidget" name="QualityMapperDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>711</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowIcon" >
   <iconset resource="qualitymapper.qrc" >
    <normaloff>:/images/qualitymapper.png</normaloff>:/images/qualitymapper.png</iconset>
  </property>
  <property name="allowedAreas" >
   <set>Qt::NoDockWidgetArea</set>
  </property>
  <property name="windowTitle" >
   <string>QualityMapperDialog</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="transferFunctionGroupBox" >
         <property name="title" >
          <string>Transfer Function</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QGraphicsView" name="transferFunctionView" >
              <property name="minimumSize" >
               <size>
                <width>350</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>350</width>
                <height>200</height>
               </size>
              </property>
              <property name="autoFillBackground" >
               <bool>true</bool>
              </property>
              <property name="verticalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="renderHints" >
               <set>QPainter::Antialiasing</set>
              </property>
              <property name="cacheMode" >
               <set>QGraphicsView::CacheBackground</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="colorbandLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="toolTip" >
               <string>Color band</string>
              </property>
              <property name="frameShape" >
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Plain</enum>
              </property>
              <property name="text" >
               <string/>
              </property>
              <property name="pixmap" >
               <pixmap resource="qualitymapper.qrc" >:/images/colorband.png</pixmap>
              </property>
              <property name="scaledContents" >
               <bool>true</bool>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QGroupBox" name="channelBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip" >
               <string>Choose the foreground channel in the Transfer Function</string>
              </property>
              <property name="title" >
               <string>Edit Channel</string>
              </property>
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QRadioButton" name="redButton" >
                 <property name="text" >
                  <string>R</string>
                 </property>
                 <property name="checked" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="greenButton" >
                 <property name="text" >
                  <string>G</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="blueButton" >
                 <property name="text" >
                  <string>B</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="selectedPointBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title" >
               <string>Selected Point</string>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QLabel" name="xLabel" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>x</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="xSpinBox" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="accelerated" >
                    <bool>true</bool>
                   </property>
                   <property name="maximum" >
                    <double>1.000000000000000</double>
                   </property>
                   <property name="singleStep" >
                    <double>0.010000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="yLabel" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>y</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="ySpinBox" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="accelerated" >
                    <bool>true</bool>
                   </property>
                   <property name="maximum" >
                    <double>1.000000000000000</double>
                   </property>
                   <property name="singleStep" >
                    <double>0.010000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QLabel" name="xQualityLabel" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>x-quality</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="invisibleQualityLabel" >
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>quality</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="presetGroupBox" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title" >
               <string>Load Preset</string>
              </property>
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QComboBox" name="presetComboBox" >
                 <property name="toolTip" >
                  <string>Transfer Functions list</string>
                 </property>
                 <property name="editable" >
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="loadPresetButton" >
                 <property name="maximumSize" >
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>click to load a Transfer Function file</string>
                 </property>
                 <property name="icon" >
                  <iconset resource="qualitymapper.qrc" >
                   <normaloff>:/images/opentf.png</normaloff>:/images/opentf.png</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="savePresetButton" >
                 <property name="maximumSize" >
                  <size>
                   <width>30</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="toolTip" >
                  <string>Add/Save preset</string>
                 </property>
                 <property name="text" >
                  <string>+</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="equalizerGroupBox" >
         <property name="title" >
          <string>Equalizer</string>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QGraphicsView" name="equalizerGraphicsView" >
              <property name="minimumSize" >
               <size>
                <width>350</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>350</width>
                <height>200</height>
               </size>
              </property>
              <property name="autoFillBackground" >
               <bool>true</bool>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="verticalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy" >
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="renderHints" >
               <set>QPainter::Antialiasing</set>
              </property>
              <property name="dragMode" >
               <enum>QGraphicsView::NoDrag</enum>
              </property>
              <property name="viewportUpdateMode" >
               <enum>QGraphicsView::SmartViewportUpdate</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QDoubleSpinBox" name="minSpinBox" >
                <property name="font" >
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="buttonSymbols" >
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="accelerated" >
                 <bool>true</bool>
                </property>
                <property name="minimum" >
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum" >
                 <double>65535.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>16</width>
                  <height>27</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" >
                <item>
                 <widget class="QDoubleSpinBox" name="midSpinBox" >
                  <property name="font" >
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="buttonSymbols" >
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="accelerated" >
                   <bool>true</bool>
                  </property>
                  <property name="minimum" >
                   <double>-65536.000000000000000</double>
                  </property>
                  <property name="maximum" >
                   <double>65535.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" >
                  <item>
                   <widget class="QLineEdit" name="midPercentageLine" >
                    <property name="maximumSize" >
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="font" >
                     <font>
                      <pointsize>10</pointsize>
                     </font>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="midPercentageLabel" >
                    <property name="text" >
                     <string>%</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0" >
                 <size>
                  <width>16</width>
                  <height>27</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="maxSpinBox" >
                <property name="font" >
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="buttonSymbols" >
                 <enum>QAbstractSpinBox::NoButtons</enum>
                </property>
                <property name="accelerated" >
                 <bool>true</bool>
                </property>
                <property name="minimum" >
                 <double>-65536.000000000000000</double>
                </property>
                <property name="maximum" >
                 <double>65535.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QGroupBox" name="groupBox_2" >
              <property name="title" >
               <string>Gamma Correction</string>
              </property>
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QLabel" name="gammaCorrectionLabel" >
                 <property name="minimumSize" >
                  <size>
                   <width>100</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize" >
                  <size>
                   <width>100</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="text" >
                  <string>gama correction view</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3" >
              <property name="title" >
               <string>Clamp Histogram</string>
              </property>
              <layout class="QHBoxLayout" >
               <item>
                <widget class="QPushButton" name="clampButton" >
                 <property name="text" >
                  <string>Clamp</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="clampHistogramSpinBox" >
                 <property name="maximum" >
                  <number>49</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="percentileLabel" >
                 <property name="text" >
                  <string>%</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox" >
              <property name="title" >
               <string>Brightness</string>
              </property>
              <layout class="QVBoxLayout" >
               <item>
                <layout class="QHBoxLayout" >
                 <item>
                  <widget class="QLabel" name="label_2" >
                   <property name="maximumSize" >
                    <size>
                     <width>10</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>0</string>
                   </property>
                   <property name="alignment" >
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="brightnessSlider" >
                   <property name="maximum" >
                    <number>100</number>
                   </property>
                   <property name="sliderPosition" >
                    <number>50</number>
                   </property>
                   <property name="orientation" >
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition" >
                    <enum>QSlider::TicksAbove</enum>
                   </property>
                   <property name="tickInterval" >
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3" >
                   <property name="maximumSize" >
                    <size>
                     <width>10</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="font" >
                    <font>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text" >
                    <string>2</string>
                   </property>
                   <property name="alignment" >
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="brightessSpinBox" >
                 <property name="font" >
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="accelerated" >
                  <bool>true</bool>
                 </property>
                 <property name="maximum" >
                  <double>2.000000000000000</double>
                 </property>
                 <property name="singleStep" >
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value" >
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="applyButton" >
           <property name="text" >
            <string>Apply</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previewButton" >
           <property name="text" >
            <string>Preview</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="resetButton" >
           <property name="text" >
            <string>Reset Histogram</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="qualitymapper.qrc" />
 </resources>
 <connections/>
</ui>
